src/WebJobs.Extensions.DurableTask/ContextInterfaces/IDurableOrchestrationClient.cs

```diff
@@ -258,8 +259,9 @@ public interface IDurableOrchestrationClient
         /// <param name="showHistory">Boolean marker for including execution history in the response.</param>
         /// <param name="showHistoryOutput">Boolean marker for including output in the execution history response.</param>
         /// <param name="showInput">If set, fetch and return the input for the orchestration instance. If both <c>showHistory</c> and <see cref = "DurableTaskOptions.StoreInputsInOrchestrationHistory" /> are also set to<c>true</c>, then the inputs for activity and sub-orchestration events in the orchestration history will also be returned.</param>
+        /// <param name="eventTypes">If set, return the history with the event types filter included.</param>
```

Comment:
I would like try a more "functional" description. What do you think of the following description for `eventTypes`?:

If provided, and `showHistory` is set, the history is filtered to contain only the types of events specified in this parameter.